docker-compose.yml

```diff
@@ -0,0 +1,112 @@
+# Create a network to use the Postgis-Server in another container.
+# Networks are used to connect containers and allow them to communicate.
+networks:
+  osm2vdv462_net:
+    name: ${DOCKER_NETWORK}
+
+# Create docker volume which is a storage point located outside of containers.
+# This is required to persistently store the database between docker restarts.
+volumes:
+  postgis:
+  ppr:
+  pgadmin:
```

Comment:
Just for clarification. Why did you add an additional volume for pgadmin?

Reply:
The Pgadmin container creates a volume by itself to permanently save contents of the folder "/var/lib/pgadmin". By creating the volume with a defined name and linking it to "pgadmin:/var/lib/pgadmin" you can get around it being named randomly by docker.
You then have three consistently named volumes (osm2vdv462_pgadmin, osm2vdv462_postgis, osm2vdv462_ppr).